src/pages/settings/project/ProjectSettings.jsx

```diff
@@ -32,27 +33,26 @@ const ProjectSettings = () => {
 
   // Global state
   const projects = useSelector(state => state.projects);
-  const selectedProject = projects.find(project => project.id === projectID);
+  let selectedProject = projects.find(project => project.id === projectID);
+  if (!selectedProject) selectedProject = {}
 
   // Derived properties
   const projectName = getProjectConfig(projects, projectID, "name");
   const secret = getProjectConfig(projects, projectID, "secret");
+  const aesKey = getProjectConfig(projects, projectID, "aesKey");
+  const contextTimeout = getProjectConfig(projects, projectID, "contextTimeout");
```

Comment:
You were supposed to take all these values from the `globalProjectConfig` right?